great_expectations/rule_based_profiler/helpers/util.py

```diff
@@ -243,36 +244,93 @@ def compute_quantiles(
         metric_values,
         q=(false_positive_rate / 2),
         axis=0,
-        interpolation="linear",  # can be omitted ("linear" is default)
     )
     upper_quantile = np.quantile(
         metric_values,
         q=1.0 - (false_positive_rate / 2),
         axis=0,
-        interpolation="linear",  # can be omitted ("linear" is default)
     )
     return lower_quantile, upper_quantile
 
 
-def compute_bootstrap_quantiles(
+def compute_bootstrap_quantiles_point_estimate(
     metric_values: np.ndarray,
     false_positive_rate: np.float64,
     n_resamples: int,
 ) -> Tuple[Number, Number]:
+    # This method is the winner of our performance testing
```

Comment:
What were the candidates? Should we possibly include that context here or is it not important?

Reply:
Good call. I have documented the candidate methods here as well now.